src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -330,8 +330,10 @@ pub struct Args {
     /// The init container for services created by the Kubernetes orchestrator.
     #[clap(long, env = "ORCHESTRATOR_KUBERNETES_INIT_CONTAINER_IMAGE")]
     orchestrator_kubernetes_init_container_image: Option<String>,
-    /// Prefix commands issued by the process orchestrator with the supplied
-    /// value.
+    /// The `StorageClass` for an ephemeral volume to be setup in `clusterd` pods.
+    /// No volume is setup if not passed.
```

Comment:
```suggestion
    /// The Kubernetes StorageClass to use for the ephemeral volume attached to
    /// services that request disk.
    ///
    /// If unspecified, the Kubernetes orchestrator will refuse to create
    /// services that request disk.
```

Reply:
(I think this one is still outstanding @ph14.)

Reply:
I'm just never going to try to use GitHub's `Commit suggestion` again 🙈